WordPress/Classes/Models/ReaderPost+Swift.swift

```diff
@@ -3,6 +3,21 @@ import WordPressUI
 
 extension ReaderPost {
 
+    /// - note: this is a workaround for https://github.com/wordpress-mobile/WordPress-iOS/issues/18975 and
+    /// https://github.com/wordpress-mobile/WordPress-iOS/issues/20953. This code
+    /// makes sure that you see the "Follow Conversation" button on P2s
+    /// but not if you have "Emails me all comments" option enabled, which
+    /// matches the behavior on the web.
+    var canActuallySubscribeToComments: Bool {
+        if canSubscribeComments {
+            return true
+        }
+        if isP2Type() {
+            return !((topic as? ReaderSiteTopic)?.emailSubscription?.sendComments ?? false)
```

Comment:
Maybe it's just me, the `!` & `?` makes it hard to read.

```suggestion
            return (topic as? ReaderSiteTopic)?.emailSubscription?.sendComments != true
```